src/client/sandbox/shadow-ui.js

```diff
@@ -56,8 +57,15 @@ export default class ShadowUI extends SandboxBase {
         return filteredList.length === nlLength ? list : filteredList;
     }
 
-    static _filterNodeList (nodeList) {
-        return ShadowUI._filterList(nodeList, item => ShadowUI._filterElement(item));
+    static _filterNodeList ({ nodeList, isLiveCollection }) {
```

Comment:
Ok.
Then you are reading `return ShadowUI._filterNodeList(nodeList, true);` you cannot understand that mean `true` parameter without navigate to definition.